lib/Controller/OutboxController.php

```diff
@@ -124,70 +106,97 @@ public function create(
 		array $bcc = [],
 		array $attachmentIds = [],
 		?int $aliasId = null,
```

Comment:
Possibly add the draft id here so the draft deletion can be triggered from here?

`$context->registerEventListener(MessageSentEvent::class, DeleteDraftListener::class);`

Is currently the event registered, changing that to a SaveOutboxMessageEvent?

Reply:
OutboxMessageCreatedEvent

Reply:
Damn it, so close. I had `OutboxMessageSavedEvent` :grin: